spec/bundle/locker_spec.rb

```diff
@@ -68,6 +80,9 @@
           ]
         EOS
         allow(locker).to receive(:`).with("brew list --versions").and_return("mysql 8.0.18")
+        allow(locker).to receive(:`).with("whalebrew list").and_return("COMMAND   IMAGE\nwget      whalebrew/wget")
+        allow(locker).to receive(:`).with("docker image inspect whalebrew/wget --format '{{ index .RepoDigests 0 }}'").and_return("whalebrew/wget@sha256:abcd1234")
```

Comment:
```suggestion
        allow(locker).to receive(:`).with("docker image inspect whalebrew/wget --format '{{ index .RepoDigests 0 }}'")
                                    .and_return("whalebrew/wget@sha256:abcd1234")
```

or whatever doesn't wrap in the GitHub diff UI